src/api/goods.ts

```diff
@@ -5,15 +5,30 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then(response => {
+      if (!response.ok) {
+        return Promise.reject(Error);
```

Comment:
Instead of returning a generic Error object, provide a specific error message. 🚩